scipy/stats/_continuous_distns.py

```diff
@@ -2900,6 +2900,129 @@ def _entropy(self, c):
 genhalflogistic = genhalflogistic_gen(a=0.0, name='genhalflogistic')
 
 
```

Comment:
Please format commit messages according to: 
http://scipy.github.io/devdocs/dev/contributor/development_workflow.html#writing-the-commit-message

Reply:
Thanks for starting to include the tags. It's close, but e.g., ideally the commit
```
`ENH GenHyperbolic`
```
might have been a few, like
```
ENH: stats: add efficient random variate sampling to GenHyperbolic
```
```
MAINT: stats: fix bug in GenHyperbolic _stats calculation
```
```
STY: stats: added whitespace for clarity in GenHyperbolic 
```

Reply:
They are better : ) But they have been showing up as 
![image](https://user-images.githubusercontent.com/6570539/104498003-0441ba00-5590-11eb-9af1-4c2a04d3b949.png)

Ideally there would be a top-level description on the first line.